site/src/load.rs

```diff
@@ -222,7 +222,7 @@ impl SiteCtxt {
             .map(|bench| {
                 (
                     bench.name.as_str().into(),
-                    Category::from_db_representation(&bench.category).unwrap(),
+                    Category::from_db_representation(&bench.category).unwrap_or(Category::Primary),
```

Comment:
I was getting panics locally without this. Let me know if this should be removed or if we should unwrap into `Category::Secondary`.